<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSwap</class>
 <widget class="QWidget" name="EditSwap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="control::MwcLineEditNormal" name="redeemAddressEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>285</y>
     <width>651</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="control::MwcLabelSmall" name="secondaryAddressLabel">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>265</y>
     <width>226</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string>Redeem address</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="addressBox">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>405</y>
     <width>661</width>
     <height>126</height>
    </rect>
   </property>
   <property name="title">
    <string>Buyer address</string>
   </property>
   <widget class="control::MwcLineEditNormal" name="destinationEdit">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>55</y>
      <width>466</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="maxLength">
     <number>1024</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="mwcmqsRadio">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>45</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MWC MQS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="torRadio">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tor</string>
    </property>
   </widget>
   <widget class="control::MwcLabelSmall" name="label_8">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>35</y>
      <width>226</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
  </widget>
  <widget class="control::MwcLabelNormal" name="tradeDesriptionLabel">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>55</y>
     <width>656</width>
     <height>196</height>
    </rect>
   </property>
   <property name="text">
    <string>Waiting for data...</string>
   </property>
  </widget>
  <widget class="control::MwcLineEditNormal" name="secondaryFeeEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>345</y>
     <width>86</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="secondaryFeeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>355</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>BTC fee:</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="secondaryFeeUnitsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>355</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>satoshi per byte</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>550</y>
     <width>116</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="startButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>545</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="stopButton">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>545</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="tradeDetailsBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>136</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Trade Details</string>
   </property>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="backButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>610</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="swapIdLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>10</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Swap ID: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>615</y>
     <width>142</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="control::MwcLabelProgress" name="progress">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="control::MwcPushButtonNormal" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>605</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Send MWC</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabelProgress.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
